src/utils/APIBreaks/APIBreak.ts

```diff
@@ -115,7 +123,10 @@ export function showAPIBreaks(version: string, ...breakInfos: APIBreakInfo[]) {
       "The following dialog boxes will provide details of the potential impact to your scripts.\n" +
       `A file with these details has also been saved on your home computer under filename ${textFileName}.`,
   );
-  details.forEach((detail, i) => {
-    dialogBoxCreate(`API BREAK VERSION ${version} DETAILS ${i + 1} of ${details.length}\n\n${detail}`);
-  });
+  for (const detail of details) {
+    if (!detail.showPopUp) {
+      continue;
+    }
+    dialogBoxCreate(`API BREAK VERSION ${version} DETAILS\n\n${detail.text}`);
```

Comment:
I liked the "1 of 6" bit, so you knew how many toasts you were sitting through. Can we keep that?